lib/tasks/locale.rake

```diff
@@ -215,4 +216,15 @@ namespace :locale do
     Rake::Task['gettext:po_to_json'].invoke
     system "rm -rf #{combined_dir}"
   end
+
+  desc "Create display names for models"
+  task "model_display_names" => :environment do
+    f = File.open(Rails.root.join("config/model_display_names.rb"), "w+")
+    Rails.application.eager_load!
+    ApplicationRecord.descendants.sort_by(&:display_name).collect do |model|
```

Comment:
Just a warning... there are some non-ApplicationRecord models.  But I do prefer this approach

Reply:
Is it ok if we're looking for descendants of `ApplicationRecord` then?

Reply:
I don't know.  If you only need a display name for things found in the database, then yes.

Reply:
Or maybe those non-ApplicationRecord classes are just required to implement the `display_name` method.  I could be okay with that until we find a better solution.